@kiva/kv-components/vue/KvToast.vue

```diff
@@ -35,13 +35,19 @@
 						"
 							:class="{
 								'tw-bg-brand tw-text-white' : (
-									messageType === '' || messageType === 'confirmation'
+									messageType === '' || messageType === 'confirmation' ||
+									messageType === 'kiva-logo'
 								),
 								'tw-bg-danger tw-text-primary-inverse' : messageType === 'error',
 								'tw-bg-caution tw-text-primary' : messageType === 'warning',
 							}"
 						>
+							<img
```

Comment:
Looks like we general inline SVGs in this repo, so that should resolve the build/test error.

Reply:
yeah, passing good now